uint8 skip_init
float32 north_angle_init
uint8 sunny_day
uint8 text_detection
uint8 bias_removal
uint8 begin_dead_reckoning
